indigo_api/models/works.py

```diff
@@ -246,10 +246,10 @@ def all_provisions(self):
 
     @cached_property
     def main_commencement(self):
-        return self.commencements.filter(main=True).first()
+        return self.commencements.filter(main=True).first() if self.pk else None
```

Comment:
Django > 4 doesn't allow relationship lookups on a model that is not saved; this tends to happen mostly in testing, so perhaps changing the test is better than this, but the tests now pass

Reply:
good to know! we might need to make this change elsewhere too